### ISSUE TITLE
Subtitle configuration bug dx

### ISSUE THREAD

**Issue by noklam** [vega/vega#2581]
# Bug
Subtitle **dx**  property is not working with {
"config": {
"title": {"dx":30}
}
but only works for
 "title": {"dx":30}}.

What is the difference of "title" and "config": {"title"} ?

# Examples
[Bug Example](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9uxQgBtMeEAEMoYRhABOKEJUKrt47nQAekttNkg6qwyqiqAtsuS37qgLQBGGAA4ATgAmOjoAdkCANjoJKNVVILAAVkwYTFUoqJZ0-xS2EBtnIwBrFUIATwAHNxAAIyMQbko4AC83KKtuTA0IOhIkVBALIcqavQBHUnV5ezg0ZW54TAU6PQsbCvLq2qmZuENiBaaQZdW9Les5A6VyzDNxdwAVG8X9Ujr5W-cAZQ+v5Q2AAklE0mGKejEhCqlGQAHo4QsEKoAHQIA6MD4ouAQOGgxjg1SIzDIrwKA6YREsFH+FE+FEAK0o0BOdkMlEwhEoKjZ3j8QVCEWisXiiUCKTSGSyOUweWSBWQAG1QCMfNwtsgfNIALrWIA)
![image](https://user-images.githubusercontent.com/18221871/80787943-71581f80-8bba-11ea-920c-e47b9347df85.png)
[Working Example](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9uxQgBtMeEAEMoYRhABOKEJUKrt46bJB1VhlVFUBbZcnOXVAWgCMMABwBOAEx06AHZvADY6CRDVVR8wAFZMGExVEJCWRM84thAzWyMAaxVCAE8ABwcQACMjEG5KOAAvBxC2M0wNCDoSJFQQAA9C0vKAR1J1eUs4NGVueEwFOj1+syKBsr0RsbhDYimakFn5vRXTOS2lQsxe8UcAFTPp-VIK+XPHAGUnl4e6fuQpMwAJJRNJhcnoxIQSpRkAB6GFTBCqAB0CC2jCeSLgEBhwMYoNU8MwiJcCi2mHhLCRniRbiRACtKNA9hZDJRMIRKCoWa4PD5-EFQuFItFvHEEkkUmlMBlYllkABtUC-NzcFbINzSAC6piAA)
![image](https://user-images.githubusercontent.com/18221871/80787965-7ddc7800-8bba-11ea-9be1-bc6a9b098baf.png)

**Comment by jheer** [comment]
@kanitw I don’t think this is a Vega issue. Vega applies dx/dy in either the title spec or title config to both the title and subtitle text. Vega also applies a title style to the title text and a separate subtitle style to the subtitle text. Applying title text styles to subtitle text would be incorrect in general. I believe Vega-Lite can use these abstractions as provided, it just needs to output to either the title config or to both the title and subtitle styles as is appropriate.
